src/views/space-board/SpaceBoard.vue

```diff
@@ -130,10 +130,18 @@ export default {
       project => project.name
     );
 
+    useInterval(
+      async () => await Promise.all([
+        loadSpaceUsers(currentSpace.value),
```

Comment:
Is there a reason to reload clouds?

Reply:
This method will reload cloud **users**.